### PR TITLE
Moving to new netflixoss build

### PR DESCRIPTION
**Pull request by rspieldenner**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -51,12 +47,6 @@ subprojects {
         )
     }
 
-    clojureTest {
```
Comment (mbossenbroek) [inline build.gradle]: Do these things that you deleted still work?

File: gradle.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-version=0.2.16-SNAPSHOT
```
Comment (mbossenbroek) [inline gradle.properties]: How would I bump the version to 0.3.0 now?

Comment (rspieldenner) [inline gradle.properties]: When this is merged we will create some jobs on cloudbees. These jobs will
let you specify if you want to bump the major minor or patch version
number. I can talk to you in more detail by showing you some of the
existing jobs.

I think what I deleted still works. Our provided plugin is setup to be
added to many configs. I would definitely run some tests and take a look to
make sure I didn't break anything though.

**Comment by mbossenbroek** [comment]
LGTM - what's the timeline for setting up cloudbees after this is merged?

**Comment by rspieldenner** [comment]
We can go and kick off the ensure and seed jobs immediately or they'll run
once a day and should be good to go tomorrow.

**Pull request merged**